src/types.rs

```diff
@@ -314,46 +316,76 @@ where
         let offset = shape.indent + 1;
         Shape::legacy(budget, offset)
     };
+
     let list_lo = context.snippet_provider.span_after(span, "(");
-    let items = itemize_list(
-        context.snippet_provider,
-        inputs,
-        ")",
-        ",",
-        |arg| arg.span().lo(),
-        |arg| arg.span().hi(),
-        |arg| arg.rewrite(context, list_shape),
-        list_lo,
-        span.hi(),
-        false,
-    );
+    let (list_str, tactic) = if inputs.len() == 0 {
```

Comment:
```suggestion
    let (list_str, tactic) = if inputs.is_empty() {
```

Reply:
Actually I also want to use is_empty, but the below error emits in this case.

```
error[E0658]: use of unstable library feature 'exact_size_is_empty' (see issue #35428)
   --> src/types.rs:321:40
    |
321 |     let (list_str, tactic) = if inputs.is_empty() {
    |                                        ^^^^^^^^
    |
    = help: add #![feature(exact_size_is_empty)] to the crate attributes to enable
```

I can fix easily this. just need to add ```add #![feature(exact_size_is_empty)].```
Can I use a feature attribute in rustfmt?

Reply:
What I'm saying is [this](https://github.com/rust-lang/rustfmt/pull/3518/commits/b7179d6b9d279e1accae47c44e50ce5e2db5cfd3)

Reply:
Sorry, I missed the comment the last time I reviewed this PR. I would rather avoid using an unstable feature in rustfmt, as it prevents updating rustfmt in the stable channel. I am sorry but could you revert the https://github.com/rust-lang/rustfmt/pull/3518/commits/b7179d6b9d279e1accae47c44e50ce5e2db5cfd3?

Reply:
Sure. I rebased it.